src/encoded/static/components/app.js

```diff
@@ -60,7 +78,7 @@ const projectList = [
     'Roadmap',
     'modENCODE',
     'modERN',
-    'GGR'
+    'GGR',
```

Comment:
JSON doesn’t allow it. AirBnB requires it if you have each value on its own line. That applies to arrays and objects.